fellowship.js

```diff
@@ -1,48 +1,111 @@
 // Dramatis Personae
 var fellowship = {
-  hobbits: [
-    "Frodo Baggins",
-    "Samwise 'Sam' Gamgee",
-    "Meriadoc \"Merry\" Brandybuck",
-    "Peregrin 'Pippin' Took"
-  ],
-  buddies: [
-    "Gandalf the Grey",
-    "Legolas",
-    "Gimli",
-    "Strider",
-    "Boromir"
-  ],
-  lands: [
-    "The Shire", "Rivendell", "Mordor"
-  ],
-  makeMiddleEarth: function(lands) {
-    console.log(lands)
-    // create a section tag with an id of middle-earth
-    // add each land as an article tag
-    // inside each article tag include an h1 with the name of the land
-    // append middle-earth to your document body
-  },
-  makeHobbits: function(hobbits) {
-    // display an unordered list of hobbits in the shire
-    // give each hobbit a class of hobbit
+    hobbits: [
+      "Frodo Baggins",
+      "Samwise 'Sam' Gamgee",
+      "Meriadoc \"Merry\" Brandybuck",
+      "Peregrin 'Pippin' Took"
+    ],
+    buddies: [
+      "Gandalf the Grey",
+      "Legolas",
+      "Gimli",
+      "Strider",
+      "Boromir"
+    ],
+    lands: [
+      "The Shire", "Rivendell", "Mordor"
+    ],
+    makeMiddleEarth: function(lands) {
+      console.log(lands)
+        // create a section tag with an id of middle-earth
+      var newElement = document.createElement('section')
+      newElement.setAttribute('id', 'middle-earth');
+      // add each land as an article tag
+      for (var i = 0; i <
+        fellowship.lands.length; i++) {
+        var newArticle = document.createElement('article')
+        var h1Tag.textContent = lands[i];
+        articleTag.appendChild(h1Tag)
+        middleEarth.appendChild(newArticle)
+      }
+      // inside each article tag include an h1 with the name of the land
+      // append middle-earth to your document body
+    },
+    makeHobbits: function(hobbits) {
+      // display an unordered list of hobbits in the shire <need to tell computer to put array of hobbits into unordered list by creating element?
+      var hobbitList = document.createElement('ul')
+        // give each hobbit a class of hobbit
+
+      for (var i = 0;) i < fellowship.hobbitts.length;
```

Comment:
try to keep each of three things on the same line when writing your for loop. Looks like a syntax error here